template.yml

```diff
@@ -223,6 +225,10 @@ Resources:
           - codepipeline:PutJobFailureResult
           - codepipeline:PutJobSuccessResult
           - apigateway:*
+          - route53:ListHostedZones
+          - route53:ChangeResourceRecordSets
+          - apigateway:*
+
           Effect: Allow
           Resource: "*"
```

Comment:
Just an idea, but for the future do you think we could do a naming pattern to restrict this to resources belonging to our project, branch or stack?

Reply:
Good idea.